ehrql/debugger.py

```diff
@@ -161,3 +194,11 @@ def related_event_columns_to_records(columns):
                     column[patient_id][row_id], convert_null=True
                 )
             yield record
+
+
+def is_ehrql_object(element):
+    isDataset = isinstance(element, Dataset)
+    isSeries = isinstance(element, BaseSeries)
+    isDateDifference = isinstance(element, DateDifference)
+    isFrame = isinstance(getattr(element, "_qm_node", None), qm.Frame)
```

Comment:
nit: rename variables in snake case (it's the python way)

Reply:
bah - yes still tripping me up from JavaScript days.